step1_SMS-PM/README.rst

```diff
@@ -0,0 +1,50 @@
+====================
+mesa/star Quickstart
```

Comment:
I'd suggest to remove this file, it looks like being just the MESA standard.
We have a README for the full project. If there is something special about the any of the steps, we should add it to the project's README.